.github/workflows/update-overcloud-host-image-tags.yml

```
@@ -0,0 +1,56 @@
+---
+name: Update overcloud host image tags
+
+on:
+  workflow_dispatch:
+    inputs:
+      rocky9_tag:
+        description: Overcloud host image tag for Rocky 9
+        type: string
+      ubuntu_noble_tag:
+        description: Overcloud host image tag for Ubuntu
+        type: string
+
+jobs:
+  propose_overcloud_host_image_tag_updates:
+    if: github.repository == 'stackhpc/stackhpc-kayobe-config'
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: write
+      pull-requests: write
+    name: Update overcloud host image tags
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: stackhpc/2025.1
+          path: ${{ github.workspace }}/src/kayobe-config
+
+      - name: Update Rocky 9 overcloud host image tag
+        run: |
+          sed -i "/stackhpc_rocky_9_overcloud_host_image_version/s/.*/stackhpc_rocky_9_overcloud_host_image_version: ${{ inputs.rocky9_tag }}/" ${{ github.workspace }}/src/kayobe-config/etc/kayobe/pulp-host-image-versions.yml
+        if: "${{ inputs.rocky9_tag != '' }}"
+
+      - name: Update Ubuntu Noble overcloud host image tag
+        run: |
+          sed -i "/stackhpc_ubuntu_noble_overcloud_host_image_version/s/.*/stackhpc_ubuntu_noble_overcloud_host_image_version: ${{ inputs.ubuntu_noble_tag }}/" ${{ github.workspace }}/src/kayobe-config/etc/kayobe/pulp-host-image-versions.yml
+        if: "${{ inputs.ubuntu_noble_tag != '' }}"
+
+      - name: Propose changes via PR if required
+        uses: peter-evans/create-pull-request@v7
+        with:
+          path: ${{ github.workspace }}/src/kayobe-config
+          commit-message: >-
+            Bump overcloud host image tags
+          author: stackhpc-ci <22933334+stackhpc-ci@users.noreply.github.com>
+          branch: bump-overcloud-host-images-${{ inputs.rocky9_tag }}-${{ inputs.ubuntu_noble_tag }}
+          delete-branch: true
+          title: >-
+            DNM test PR: Bump overcloud host image tags 
```

This should be a real PR right?
```suggestion
            Bump overcloud host image tags 
```